.github/ISSUE_TEMPLATE/translation_issue.yml

```diff
@@ -1,79 +1,117 @@
 name: "🌐 Localization/Translation issue"
-description: Report incorrect translations.
-labels:
-- Issue-Bug
-- Area-Localization
-- Issue-Translation
-- Needs-Triage
+description: Report incorrect translations
+labels: [Needs-Triage, Issue-Bug, Issue-Translation, Area-Localization]
+assignees: crutkas
```

Comment:
remove

Reply:
Why? As far as I know (let me know if I'm wrong) you're the **only** link between this repo and the translation team? That makes it "by definition" your task.